packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -85,6 +90,10 @@ export default class UpdateConfigure extends EasCommand {
       skipSDKVersionRequirement: true,
     });
 
+    if (!isExpoUpdatesInstalledOrAvailable(projectDir, exp.sdkVersion)) {
+      await installExpoUpdatesAsync(projectDir);
```

Comment:
Shouldn't we display a prompt asking the user if they want to install expo-updates? I'm not telling to do so but I'm curious.

Reply:
I considered this, but since they are running `update:configure` it seemed like their answer would always be yes unless they were running the wrong command altogether. And in those rare cases it is easy to fix.

So it seemed like average developer speed would be improved by not asking them a redundant question.